gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/InMemoryGoogleCloudStorage.java

```diff
@@ -542,4 +544,15 @@ public GoogleCloudStorageItemInfo composeObjects(
     destChannel.close();
     return getItemInfo(destination);
   }
+
+  /**
+   * Get the Value of the statistics
+   *
+   * @param key name of the object related statistics key
+   * @return
+   */
+  @Override
+  public AtomicLong getObjectStatistics(GoogleCloudStorageStatistics key) {
+    return new AtomicLong(0);
```

Comment:
we should either track statistics or throw UnsupportedOperationException here

Reply:
Updated